### PR TITLE
Fix footer if the content is too small

### PR DESCRIPTION
**Pull request by ayrat555**
before 
Screenshot 2021-09-15 at 09 16 03

after
Screenshot 2021-09-15 at 09 54 56

### CODE REVIEW

File: sass/juice.scss [hunk 1]
```diff
@@ -7,6 +7,10 @@ body {
   margin: 0;
   box-sizing: border-box;
   background-color: var(--secondary-color);
+  justify-content:space-between;
+  display:flex;
+  flex-direction:column;
```
Comment (Folyd) [inline sass/juice.scss]: I'd like to keep the `body` less styled as possible, and the `min-height` is enough to fix this. What do you think?

**Review by Folyd — CHANGES_REQUESTED** [review]
Thanks for the PR.  Can we remove the `flex` style?

File: sass/juice.scss (hunk 1 above)
Comment (ayrat555) [reply sass/juice.scss]: I tested locally,the behaviour is the same (wrong) without flex

Comment (Folyd) [reply sass/juice.scss]: You are right.  I have another idea to fix this:
```diff
.body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
}

.main {
+   flex-grow: 1;
}

.content {
-   justify-content: center;
}
```

Comment (ayrat555) [reply sass/juice.scss]: @Folyd I did as you suggested

**Review by Folyd — APPROVED** [review]

**Pull request merged**